package.json

```diff
@@ -9,11 +9,12 @@
     "storybook": "start-storybook -p 9001 -c .storybook"
   },
   "dependencies": {
+    "@skatejs/val": "^0.4.0",
+    "@storybook/addon-options": "^4.1.13",
```

Comment:
This is not a dependency. Is a devDependecy cause our consumer do not have to install it

Reply:
Fine. Done.